src/ReleaseHistory.md

```diff
@@ -5,6 +5,7 @@
 * FEATURE: Allow emitting non-failure tool notifications as debug/informational messages.
 * FEATURE: `SarifLogger` now populates `tool.driver`'s `organization` and `product` properties instead of adding `"Company"` and `"ProductName"` to `tool.driver'`s property bag. [#1716](https://github.com/microsoft/sarif-sdk/issues/1716)
 * FEATURE: Add `closeWriterOnDispose` argument (with a default of 'true') that indicates whether SarifLogger writers are closed during its Dispose() method. Providing a value of `false` to this argument allows SarifLogger to work against a stream that can subsequently be reused (for example, to deserialize the logged content back to a `SarifLog` instance).
+* BUGFIX: Update PREfast converter to handle paths with no trailing slash and the inclusion of optional suppression data.
```

Comment:
>suppression [](start = 104, length = 11)

NIT: I'd've split this in two and characterized the inclusion of suppression data as a feature.